frontend/packages/knative-plugin/src/components/test-function/types.ts

```diff
@@ -3,17 +3,30 @@ export enum InvokeFormat {
   CloudEvent = 'CloudEvent',
 }
 
-export enum HTTPMessageTypes {
+export enum ModalPanel {
   Request = 'Request',
   Response = 'Response',
 }
 
-export enum RequestPaneTab {
-  Body = 'Body',
-  Options = 'Options',
-}
-
-export enum ResponsePaneTab {
-  Body = 'Body',
-  Info = 'Info',
-}
+export type TestFunctionFormikValues = {
+  request: {
+    format: InvokeFormat;
+    contentType: string;
+    type: string;
+    source: string;
+    customHeaders: string[][];
+    body: {
+      data: string;
+    };
+  };
+  response: {
+    status: string;
+    statusCode: number;
+    header: Record<string, string[]>;
+    body: string;
+  };
+  endpoint: {
+    url: URL | string;
+    private: boolean;
+  };
```

Comment:
Isn't used anymore, can you please remove it.

Reply:
Done :)